test/Atmos/EDMF/closures/pressure.jl

```diff
@@ -0,0 +1,57 @@
+#### Pressure model kernels
+
+"""
+    perturbation_pressure(
+        m::AtmosModel{FT},
+        press::PressureModel,
+        state::Vars,
+        diffusive::Vars,
+        aux::Vars,
+        t::Real,
+        i,
+    ) where {FT}
+Returns the value of perturbation pressure gradient
+for updraft i, as well as the corresponding source term
+in the environmental TKE budget following He et al. (JAMES, 2020),
+given:
+ - `m`, an `AtmosModel`
+ - `press`, a `PressureModel`
+ - `state`, state variables
+ - `diffusive`, additional variables
+ - `aux`, auxiliary variables
+ - `t`, the time
+ - `i`, index of the updraft
+"""
+function perturbation_pressure(
+    m::AtmosModel{FT},
+    press::PressureModel,
+    state::Vars,
+    diffusive::Vars,
+    aux::Vars,
+    t::Real,
+    i,
+) where {FT}
+
+    # Alias convention:
+    gm = state
+    en = state.turbconv.environment
+    up = state.turbconv.updraft
+    up_a = aux.turbconv.updraft
+    up_d = diffusive.turbconv.updraft
+
+    ρinv = 1 / gm.ρ
+    N_up = n_updrafts(m.turbconv)
+    en_area = environment_area(state, aux, N_up)
+    w_env = environment_w(state, aux, N_up)
+    w_up = up[i].ρaw / up[i].ρa
+
```

Comment:
The lines above are repeated in many parameterization functions. Would be good to have some wrapper, similar to thermo_state, to populate everything.

Reply:
I agree. I think we can do this with the env vars. I'll try to implement what I had in mind.

Reply:
I've added `environment_vars`, which computes the environment `a` and `w`. This is passed to quite a few places.

Reply:
Let's leave `w_up_i` on the fly, since this doesn't require a loop over updrafts.